tests/admin_views/tests.py

```diff
@@ -1739,6 +1739,32 @@ def test_history_view(self):
 
             self.client.get(reverse('admin:logout'))
 
+    def test_history_view_verbose_model_name(self):
+        login_url = '%s?next=%s' % (
```

Comment:
one line is fine here (prefer longer lines up to 119 chars when it improves readability)
don't think `?next=%s` is needed